### ISSUE TITLE
Filter by genre, year,... 

### ISSUE THREAD

**Issue by Jordigb44** [jarnedemeulemeester/findroid#854]
Hello, i love your job, it is amazing, but I would like to know if you would add the option to filter by film genre, film release year,.. Becouse I think this is a hopefully feature to easy find your preferred type of films. Thanks for all your great job

**Comment by spoo333** [comment]
Yes please, a filter by genre etc option would be fantastic. Thank you for making findroid.

**Comment by Crane2222** [comment]
Bumping this request, amazed there haven't been more requests for this feature. Native web/app supports this so shouldn't be too tricky?
